app/Main.js

```diff
@@ -25,7 +25,10 @@ try {
 function runApp(initial_state) {
     console.log('Initial state', initial_state);
     const config = initial_state.offchain.config
-    steem.config.set('websocket', config.ws_connection_client);
+    if(process.env.USE_JSONRPC)
```

Comment:
I see what you mean @sneak  - I'm not 100% on that, but I'll adjust and have it pull that from the config object instead of process.env - that should be sufficient